cpp/ps0005/cpp_ps0005_solution .cpp

```diff
@@ -1,70 +1,83 @@
-
-
-#include<iostream>
+#include <iostream>
 using namespace std;
+
 int main()
 {
+    // prompting user for 2 operands and an operator 2/2
     int x;
     double num1, num2, result;
 
-
-    cout << "enter the operator \n";
-
-
     cout << "1. (+)\n";
     cout << "2. (-)\n";
     cout << "3. (*)\n";
-    cout << "4.(/)\n";
+    cout << "4. (/)\n";
     cout << "5. (%)\n";
     cout << "enter the number of proces ";
-
     cin >> x;
 
     cout << "enter two numbers:\n ";
     cin >> num1 >> num2;
+

```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input